pages/success/content/index.js

```diff
@@ -105,21 +105,21 @@ module.exports = {
       title: 'Approved'
     },
     whatNext: {
-      body: `This request has been approved.`,
+      body: ``,
       internal: 'The relevant people have been emailed.',
       external: ''
     },
     taskLink: {
-      before: 'You can ',
-      linkText: 'view the history of this request.'
+      before: '',
```

Comment:
removing `You can` is this intentional?

Reply:
yes, a copy change requested by UCD team